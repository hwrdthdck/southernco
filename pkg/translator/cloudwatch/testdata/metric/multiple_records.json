[
  {
    "metric_stream_name":"MyMetricStream",
    "account_id":"1234567890",
    "region":"us-east-1",
    "namespace":"AWS/EC2",
    "metric_name":"DiskWriteOps",
    "dimensions":{
      "InstanceId":"i-123456789012"
    },
    "timestamp":1611929698000,
    "value":{
      "count":3.0,
      "sum":20.0,
      "max":18.0,
      "min":0.0,
      "p99":17.56,
      "p99.9":17.8764,
      "TM(25%:75%)":16.43
    },
    "unit":"Seconds"
  },
  {
    "metric_stream_name":"MyMetricStream",
    "account_id":"1234567890",
    "region":"us-east-1",
    "namespace":"AWS/EC2",
    "metric_name":"DiskWriteOps",
    "dimensions":{
      "InstanceId":"i-123456789012"
    },
    "timestamp":1611929698100,
    "value":{
      "count":5.0,
      "sum":21.0,
      "max":18.0,
      "min":0.0,
      "p99":17.56,
      "p99.9":17.8764,
      "TM(25%:75%)":16.43
    },
    "unit":"Seconds"
  }
]