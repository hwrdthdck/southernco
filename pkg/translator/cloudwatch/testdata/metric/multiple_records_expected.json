{
  "resourceMetrics":[
    {
      "resource":{
        "attributes":[
          {
            "key":"cloud.provider",
            "value":{
              "stringValue":"aws"
            }
          },
          {
            "key":"cloud.account.id",
            "value":{
              "stringValue":"1234567890"
            }
          },
          {
            "key":"cloud.region",
            "value":{
              "stringValue":"us-east-1"
            }
          },
          {
            "key":"service.namespace",
            "value":{
              "stringValue":"AWS"
            }
          },
          {
            "key":"service.name",
            "value":{
              "stringValue":"EC2"
            }
          },
          {
            "key":"aws.cloudwatch.metric_stream_name",
            "value":{
              "stringValue":"MyMetricStream"
            }
          }
        ]
      },
      "scopeMetrics":[
        {
          "scope":{

          },
          "metrics":[
            {
              "name":"DiskWriteOps",
              "unit":"Seconds",
              "summary":{
                "dataPoints":[
                  {
                    "attributes":[
                      {
                        "key":"service.instance.id",
                        "value":{
                          "stringValue":"i-123456789012"
                        }
                      }
                    ],
                    "timeUnixNano":"1611929698000000000",
                    "count":"3",
                    "sum":20,
                    "quantileValues":[
                      {
                        "quantile":99,
                        "value":17.56
                      },
                      {
                        "quantile":99.9,
                        "value":17.8764
                      },
                      {

                      },
                      {
                        "quantile":100,
                        "value":18
                      }
                    ]
                  },
                  {
                    "attributes":[
                      {
                        "key":"service.instance.id",
                        "value":{
                          "stringValue":"i-123456789012"
                        }
                      }
                    ],
                    "timeUnixNano":"1611929698100000000",
                    "count":"5",
                    "sum":21,
                    "quantileValues":[
                      {
                        "quantile":99,
                        "value":17.56
                      },
                      {
                        "quantile":99.9,
                        "value":17.8764
                      },
                      {

                      },
                      {
                        "quantile":100,
                        "value":18
                      }
                    ]
                  }
                ]
              }
            }
          ]
        }
      ]
    }
  ]
}